src/Chainweb/Pact/Service/PactInProcApi.hs

```diff
@@ -89,15 +89,34 @@ withPactService'
     -> (PactQueue -> IO a)
     -> IO a
 withPactService' ver cid logger memPoolAccess bhDb pdb sqlenv config action = do
-    reqQ <- atomically $ newPactQueue (_pactQueueSize config)
-    race (server reqQ) (client reqQ) >>= \case
+    reqQ <- newPactQueue (_pactQueueSize config)
+    race (concurrently_ (monitor reqQ) (server reqQ)) (client reqQ) >>= \case
         Left () -> error "pact service terminated unexpectedly"
         Right a -> return a
   where
     client reqQ = action reqQ
     server reqQ = runForever logg "pact-service"
         $ PS.initPactService ver cid logger reqQ memPoolAccess bhDb pdb sqlenv config
     logg = logFunction logger
+    monitor = runPactServiceQueueMonitor $ addLabel ("sub-component", "PactQueue") logger
+
+runPactServiceQueueMonitor :: Logger logger => logger ->  PactQueue -> IO ()
+runPactServiceQueueMonitor l pq = do
+  let lf = logFunction l
+  logFunctionText l Info "Initialized PactQueue"
+  runForeverThrottled lf "Chainweb.Pact.Service.PactInProcApi.runPactServiceQueueMonitor" 10 (10 * mega) $ do
+        (validateblock_stats, newblock_stats, other_stats) <- getPactQueueStats pq
+        logFunctionText l Debug "got validateBlock stats"
+        logFunctionJson l Info validateblock_stats
+        logFunctionText l Debug "logged validateBlock stats"
+        logFunctionText l Debug "got newBlock stats"
+        logFunctionJson l Info newblock_stats
+        logFunctionText l Debug "logged newBlock stats"
+        logFunctionText l Debug "got otherMsg stats"
+        logFunctionJson l Info other_stats
+        logFunctionText l Debug "logged otherMsg stats"
```

Comment:
I wonder if those `Debug` logs are a bit too excessive. Maybe just remove those? Or just condense all of them into a single `Debug` message?